<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatForm</class>
 <widget class="QWidget" name="StatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="7">
    <widget class="QPushButton" name="mUpdateBtn">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>End Day</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QComboBox" name="mUnitCombo"/>
   </item>
   <item row="0" column="3">
    <widget class="QDateEdit" name="mEndDate"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start Day</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="mStartDate"/>
   </item>
   <item row="1" column="0" colspan="8">
    <layout class="QGridLayout" name="mGridLayout"/>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
